tools/gfastats/gfastats.xml

```diff
@@ -56,9 +54,14 @@
                 --seq-report
                 $mode_condition.statistics_condition.out_sequence
             #end if
-            $mode_condition.tabular > '$stats' 
+            $mode_condition.tabular > '$stats'
+        #else
+            --agp-to-path $mode_condition.agp_to_path
+            $mode_condition.discover_paths
+            -o dataset.gfa
         #end if
-        #if $mode_condition.selector == 'manipulation'
+        #if $mode_condition.selector == 'manipulation' or $mode_condition.selector == 'scaffolding'
+            && ls -lah
```

Comment:
That looks like a debug statement

Reply:
It was removed in the last commit I think.